planner/core/point_get_plan.go

```diff
@@ -166,23 +168,30 @@ func (p *PointGetPlan) AccessObject(normalized bool) string {
 
 // OperatorInfo implements dataAccesser interface.
 func (p *PointGetPlan) OperatorInfo(normalized bool) string {
-	buffer := bytes.NewBufferString("")
+	var buffer strings.Builder
```

Comment:
Can we do the same thing to other `OperatorInfo`?

Reply:
If our buffer is or will be used as string, it's better to switch to `strings.Builder`.

Reply:
move to line 177.